Makefile

```diff
@@ -9,6 +14,7 @@ upload_% : merged/%.pdf
 	aws s3 cp $< s3://$$S3_BUCKET_NAME --acl public-read
 
 merged/%.pdf : $(addsuffix .pdf,$(basename $(ATTACHMENTS)))
+	$(foreach attachment,$^,$(call decrypt,$(attachment)))
```

Comment:
Generally, I like to keep processing steps separate, however it felt a lot more natural to add decryption to the merge recipe because I'm updating the dependencies inplace, and it doesn't really make sense to me to define an intermediate recipe that doesn't create new files. I'm open to separating it out, then adding that intermediate step as the dependency here, but my preference is for the tighter option.